quiche/src/recovery/gcongestion/bbr2/network_model.rs

```diff
@@ -611,12 +618,15 @@ impl BBRv2NetworkModel {
             return true;
         }
 
-        self.rounds_without_bandwidth_growth += 1;
+        let ignore_round = self.ignore_app_limited_for_no_bandwidth_growth &&
+            congestion_event.last_packet_send_state.is_app_limited;
+
+        if !ignore_round {
+            self.rounds_without_bandwidth_growth += 1;
+        }
 
         // full_bandwidth_reached is only set to true when not app-limited
-        if self.rounds_without_bandwidth_growth >= params.startup_full_bw_rounds &&
-            !congestion_event.last_packet_send_state.is_app_limited
-        {
+        if self.rounds_without_bandwidth_growth >= params.startup_full_bw_rounds {
```

Comment:
But alert: we lost the `&& !congestion_event.last_packet_send_state.is_app_limited` in the next line.

That was fine when `self.rounds_without_bandwidth_growth` was only incremented when not appl limited, but that's no longer the case with the introduction of config.